.github/actions/release-alpha/action.yml

```diff
@@ -15,8 +15,10 @@ runs:
         git config --local user.email "x@empathy.co"
         git config --local user.name "empathy/x"
       shell: bash
-    - name: Create .npmrc file with authorization token
-      run: echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > .npmrc
+    - name: Configure npm auth
+      run: |
+        npm config set registry="http://registry.npmjs.org/"
```

Comment:
```suggestion
        npm config set registry="//registry.npmjs.org/"
```

Reply:
It is needed to specify the registry, anyway I'll use https :sisi3: